schemas/theme/translations_schema.json

```diff
@@ -0,0 +1,57 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "description": "A schema for Shopify theme translation files. Supports nested properties, pluralized translations, and translations containing HTML.",
+  "type": "object",
+  "additionalProperties": {
+    "oneOf": [
```

Comment:
I believe we could change this line to avoid the following message

```suggestion
    "anyOf": [
```

![image](https://github.com/Shopify/theme-liquid-docs/assets/1079279/21bc1bd7-8279-4af4-8dcc-f29742483422)